### ISSUE TITLE
LaTeX versions

### ISSUE THREAD

**Issue by rljacobson** [RuleBasedIntegration/IntegrationRules#1]
You have the original [Mathematical notebook sources](https://github.com/RuleBasedIntegration/Rubi/tree/master/IntegrationRuleNotebooks) from which these PDFs are derived. I took a brief look at the notebook sources, and it seems to me that it should be possible to auto-generate LaTeX sources for these rules. A LaTeX version would be more useful for some applications and could be compiled to PDF, Markdown+MathJax, HTML, etc.

**Comment by halirutan** [comment]
Yes, I agree that is possible. I always hope someone jumps on board who could work on this. I know several people from SE who already have nb -> markdown converters who basically do exactly this. With something like this, we could create the content for a webpage directly instead of using PDF files.

There is a downside though. Some of the rule notebooks are pretty large and contain many formulas. I'm not sure how performant MathJax will be if it is confronted with a [20 page long](https://rulebasedintegration.org/PdfRuleFiles/1%20Algebraic%20functions/1.1%20Binomial%20products/1.1.1%20Linear/1.1.1.2%20(a+b%20x)%5Em%20(c+d%20x)%5En.pdf) text that contains many many things to render.

Note that I updated this repo some minutes ago to represent Rubi engine version 4.16.0. With it, I included the code for creating the PDF files and the README automatically [here](https://github.com/RuleBasedIntegration/IntegrationRules/blob/master/RubiPDFTools.m).

**Comment by rljacobson** [comment]
KaTeX might work: https://katex.org/. It is not as bulletproof and full-featured as MathJax, but for most simple math expressions it works well, very quickly, and without page reflow.

Nice low-hanging fruit for a new contributor during the next [Hacktoberfest](https://hacktoberfest.digitalocean.com/)!
